### PR TITLE
Integral variant for Regge/HHJ 

### PR DESCRIPTION
**Pull request by pbrubeck**
This enables HHJ in 3D

### CODE REVIEW

File: FIAT/hellan_herrmann_johnson.py [hunk 1]
```diff
@@ -3,79 +3,100 @@
...
-            pts = ref_el.make_points(dim, entity_id, degree + 2)
-            normal = ref_el.compute_scaled_normal(entity_id)
-            nodes.extend(InnerProduct(ref_el, normal, normal, pt) for pt in pts)
-            num_new_nodes = len(pts)
-            entity_ids[entity_id] = list(range(offset, offset + num_new_nodes))
-            offset += num_new_nodes
-        return nodes, entity_ids
+        # Interior dofs
+        cur = len(nodes)
+        if sd == 2 and variant == "point":
```
Comment (rckirby) [inline FIAT/hellan_herrmann_johnson.py:53]: Do we want to catch the case where `variant` is other than `"point"` or `"integral"` as an exception?

File: test/unit/test_fiat.py [hunk 2]
```diff
@@ -263,6 +263,9 @@ def __init__(self, a, b):
     "HellanHerrmannJohnson(T, 0)",
     "HellanHerrmannJohnson(T, 1)",
```
Comment (rckirby) [inline test/unit/test_fiat.py:264]: Do we have any tests for the variants?

**Review by rckirby — APPROVED** [review]
Looks good, just a few questions

File: FIAT/hellan_herrmann_johnson.py (hunk 1 above)
Comment (pbrubeck) [reply FIAT/hellan_herrmann_johnson.py:53]: that's handled by `check_format_variant`

**Pull request merged**